src/dmd/dscope.d

```diff
@@ -66,6 +66,8 @@ enum SCOPE
     // The following are mutually exclusive
     printf        = 0x4_0000, /// printf-style function
     scanf         = 0x8_0000, /// scanf-style function
+
+    subfuncscope  = 0x100_000, /// in a function and in a compound scope
```

Comment:
It might seem counterintuitive, but it would be good to have the underscore in the same position as the values above (`0x10_0000`).